fireant/slicer/queries/builder.py

```diff
@@ -195,8 +195,9 @@ def fetch(self, hint=None) -> Iterable[Dict]:
         # Apply operations
         operations = find_operations_for_widgets(self._widgets)
         for operation in operations:
-            df_key = format_metric_key(operation.key)
-            data_frame[df_key] = operation.apply(data_frame)
+            for reference in [None] + self._references:
```

Comment:
What is the reason for appending a None to the references?

Reply:
you'll see this in a lot of places in the code. need to apply the operation to the original data and then for each reference. The `None` is, like the code reads, the first value for reference while iterating through the references.

Reply:
ok, thanks!